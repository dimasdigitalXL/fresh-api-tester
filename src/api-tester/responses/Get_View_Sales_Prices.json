{
  "data": [
    {
      "id": "501",
      "customer": null,
      "customerGroup": {
        "id": "6",
        "name": "Händler",
        "reference": "HK"
      },
      "validFrom": null,
      "expiresAt": null,
      "amount": 1,
      "price": {
        "amount": "10.00000000",
        "currency": "EUR"
      }
    },
    {
      "id": "474",
      "customer": null,
      "customerGroup": {
        "id": "6",
        "name": "Händler",
        "reference": "HK"
      },
      "validFrom": null,
      "expiresAt": null,
      "amount": 100,
      "price": {
        "amount": "7.50000000",
        "currency": "EUR"
      }
    },
    {
      "id": "472",
      "customer": null,
      "customerGroup": null,
      "validFrom": null,
      "expiresAt": null,
      "amount": 1,
      "price": {
        "amount": "0.18000000",
        "currency": "EUR"
      }
    },
    {
      "id": "473",
      "customer": null,
      "customerGroup": null,
      "validFrom": null,
      "expiresAt": null,
      "amount": 2,
      "price": {
        "amount": "1.80000000",
        "currency": "EUR"
      }
    },
    {
      "id": "515",
      "customer": {
        "id": "2",
        "name": "Karlo Kunde GmbH & Co. KG"
      },
      "customerGroup": null,
      "validFrom": null,
      "expiresAt": null,
      "amount": 1,
      "price": {
        "amount": "5.00000000",
        "currency": "EUR"
      }
    }
  ],
  "extra": {
    "page": {
      "number": 1,
      "size": 10
    }
  }
}
