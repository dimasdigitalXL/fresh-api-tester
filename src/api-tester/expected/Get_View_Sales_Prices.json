{
  "data": [
    {
      "id": "string",
      "customer": null,
      "customerGroup": {
        "id": "string",
        "name": "string",
        "reference": "string"
      },
      "validFrom": null,
      "expiresAt": null,
      "amount": 0,
      "price": {
        "amount": "string",
        "currency": "string"
      }
    }
  ],
  "extra": {
    "page": {
      "number": 0,
      "size": 0
    }
  }
}
